docs/developer/implement-a-dashboard-plugin.md

```diff
@@ -0,0 +1,105 @@
+---
+sidebar_label: Implementing a plugin
```

Comment:
Do you know where is the top-level sidebar text set? e.g. "Dashboard Plugins". Looking at capture-app, I see that the `sidebar_label` is the menu under "Capture Plugins"